userguide/tutorials/using-https.adoc

```diff
@@ -0,0 +1,126 @@
+= Using HTTPS and SSL/TLS
+
+== Overview
+
+The HTTP protocol, commonly used for message exchange over the internet, does not provide encryption. As a result messages exchanged with this protocol may be insecure and subject to eavesdropping, and their source may be uncertain. To avoid this problem, HTTP is frequently combined with an encryption scheme known as SSL, or more recently TLS, that can be used to keep messages safe from unauthorized access, and verify that the sender is who they claim to be. This secure protocol is called HTTPS. 
+
+HTTPS is based on the use of a verification document called a *certificate*. the certificate serves two purposes:
+
+1. It provides the keys necessary for a highly secure encryption.
+2. It provides the electronic signature of a *certificate authority (CA)* who guarantees the identity of the sender.
+
+The details of the structure of certificates are defined by international standards and are well beyond the scope of this discussion. One thing you need to understand, though, is the certificate authority. Normally this is either an organization that is intrinsically well-known and trusted, such as DigiCert or GlobalSign (called a *root* CA), or an organization in the *trust hierarchy* derived from a root. A CA in a trust hierarchy must be itself certified by another CA in a chain which eventually leads to the root CA. Your CA, in turn, must be able to certify that *you* are who you say you are!
+
+For temporary use, you can sometimes create a *self-signed* certificate, in which you are your own CA. Most browsers will frown on this, since you are simply vouching for yourself, but it allows you to get started with security.
+
+If you own a domain, your domain provider can serve as your CA. If you don't have a domain, you can purchase one for very low cost from a domain provider such as Godaddy. When you obtain a domain you need to prove your identity, so your domain provider must trust you. To demonstrate this trust you will need a CNAME.
+
+== CNAMEs
+
+A CNAME, or *Canonical Name*, is an identifier that links your CA to one or more resources that you are protecting with your certificate.
+
+== Obtaining a Certificate
+
+There are many ways to obtain a certificate that you can use to protect a particular resource. Here we will describe three that may be of use in particular Kill Bill scenarios, especially those running on AWS. These are:
+
+1. Using a free CA (such as *Let's Encrypt*)
+2. Using the AWS Certificate Manager
```

Comment:
If I understand correctly, you have only discussed option 1. below -- are you planning to add `2.` and `3.` as well, and then refer the various doc to point the relevant sections?